mkdocs.yml

```diff
@@ -11,144 +11,146 @@ markdown_extensions:
   - mdx_math
   - footnotes
 nav:
-- Home: index.md
```

Comment:
Sorry for bigger diff due to formatting. I can split it to another PR if that would be more handy.